components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -1064,3 +1065,112 @@ func (backend *ESClient) SetDailyLatestToFalseForControlIndex(ctx context.Contex
 		Do(ctx)
 	return errors.Wrap(err, "SetDailyLatestToFalseForControlIndex")
 }
+
+func (backend *ESClient) GetNodesDayLatestTrue(ctx context.Context) ([]relaxting.NodesUpgradation, error) {
```

Comment:
please add the method descriptions for all new methods.

Reply:
done